pkg/backup/item_backupper.go

```diff
@@ -317,8 +323,32 @@ func (ib *itemBackupper) executeActions(
 			continue
 		}
 		log.Info("Executing custom action")
+		if ib.backupRequest.ResPolicies != nil && groupResource.String() == "persistentvolumeclaims" && action.Name() == "velero.io/csi-pvc-backupper" {
```

Comment:
Two suggestions here:
1. Check the action name may not be a good choice:

    - The action name is unique for each plugin, multiple plugins could do the same task, i.e., take CSI snapshot. If we check the action name, every plugin will need to add its name at here. Specifically, the current code could only filter out the action from Velero's CSI plugin
    - The action name is actually the plugin name, plugin could change this any time. Whenever it is changed, the code here need to be changed
2. Let's wrap these checking code into a function, which helps reduce the complexity of the main workflow

Reply:
I'll write one item in the release notes to clarify that current resource policies are not working for third-party plugins